sormas-api/src/main/resources/captions.properties

```diff
@@ -358,6 +359,7 @@ caseSearchSpecificCase=Search specific case
 caseSearchCase=Search case
 caseSelect= Select case
 caseCreateNew=Create new case
+caseReferenceDefinition=Reference definition
```

Comment:
Since this is a new actual field, please add it as a proper caption (=> CaseData.caseReferenceDefinition)

Reply:
updated